HW_1/task_3.py

```diff
@@ -0,0 +1,36 @@
+"""
+Напишите программу, которая принимает на вход координаты точки (X и Y) и выдаёт номер четверти плоскости,
+в которой находится эта точка (или на какой оси она находится).
+
+Ввод: два значения типа <int>
+Вывод: значение типа <int> либо значение типа <str>
+
+Пример:
+
+34
+-30
+4
+
+2
+4
+1
+
+-34
+0
+Точка на отрицательной части оси абсцисс
+"""
+
+x = int(input('Введите координату точки x, x≠0:'))
+y = int(input('Введите координату точки y, y≠0:'))
+
+if x == 0 and y == 0:
```

Comment:
Если в условии присутствуют повторяющееся значение, операторы сравнения, равенства и оператор `and`, то условия можно свернуть относительно повторяющегося значения.
```suggestion
if x == y == 0:
```